Cargo.toml

```diff
@@ -1,8 +1,13 @@
 [package]
 name = "fsrs"
 version = "0.1.0"
+authors = ["Open Spaced Repetition"]
 edition = "2021"
+keywords = ["spaced-repetition", "algorithm", "fsrs", "machine-learning"]
 license = "BSD-3-Clause"
+readme = "README.md"
+repository = "https://github.com/open-spaced-repetition/fsrs-rs"
+description = "FSRS for Rust, including Optimizer and Scheduler"
```

Comment:
add `categories` and `rust-version` ? 

ref https://crates.io/categories and https://doc.rust-lang.org/cargo/reference/manifest.html#the-categories-field